test/test.go

```diff
@@ -145,14 +144,24 @@ func ParseTestResults(testsRootDir string) (results Results, err error) {
 
 		// find step-info in dir, continue if no step-info.json as this file is only required if step has exported artifacts also
 		// <root_tests_dir>/<test_dir>/step-info.json
+
+		stepInfoPth := filepath.Join(testDirPath, "step-info.json")
+		if isExists, err := pathutil.IsPathExists(stepInfoPth); err != nil {
+			log.Warnf("Failed to check if step-info.json file exists in dir: %s: %s", testDirPath, err)
+			continue
+		} else if !isExists {
+			continue
+		}
```

Comment:
This checks looks much nicer than just trying to immediately read the `step-info.json` file and print out the `Failed to read step-info.json file in dir: %s, error: %s` failure. 

I am just wondering if did you run into this as you were testing your feature? Because it looks a bit out of place when I am looking at the rest of the PR. 😄

Reply:
Yeah, sorry I haven't mention this in the PR description, I was pretty unhappy with the current log structure and wanted to achieve this log layout:
```
[info] Collecting xyz...
[normal] details

[info] Deploying xyz...
[normal] details
```

That's why I started improving the test result deployment logs, and I faced this `log.Debugf(colorstring.Yellowf`, which seemed to me as a warning log.

I changed it to a warning log, but then I realised there is always 1 directory, withouth test report related files (the test report dir prepared for this step run, the CLI will clean it up after the step run, but when collecting all the test results, it is there) and so that the code always would have print this warning.

Now I check if the required test report file exists and if not, I just continue the loop.